tests/autorestart_test/test_container_autorestart.py

```diff
@@ -37,8 +38,12 @@ def get_process_id(duthost, container_name, process):
     """
     running_status = False
     process_id = -1
-    ps_out = duthost.shell("docker exec {} ps -ax".format(container_name))
-    for line in ps_out["stdout_lines"]:
+    is_running = duthost.shell("docker inspect -f \{\{.State.Running\}\} %s" % container_name)
```

Comment:
Suggest to use newer `.format()` syntax rather than `%`. You are using `.format()` in other areas of the file; please be consistent.

Reply:
Initially, I want to use `.format()` to keep consistent. But I came across with an error `KeyError`. So I followed the example :the line 298 at devices.py.

Reply:
Sounds like your syntax was incorrect. I believe this should work:

```
is_running = duthost.shell("docker inspect -f {{{{.State.Running}}}} {}".format(container_name))
```

Reply:
Yes, It did work.

Reply:
The usage of syntax `\{\{.State.Running\}\}` is not correct.

Reply:
Exactly.